packages/eslint-plugin-one-app/package-lock.json

```diff
@@ -423,501 +423,6 @@
 			"integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
 			"dev": true
 		},
-		"fsevents": {
```

Comment:
can you remove the changes to `package-lock` from this PR ? not sure why this would have changed